src/Element_TEST.cc

```diff
@@ -72,6 +72,18 @@ TEST(Element, Required)
   ASSERT_EQ(elem.GetRequired(), "1");
 }
 
+/////////////////////////////////////////////////
+TEST(Element, SetSet)
```

Comment:
Can you add an integration test where a file is loaded using `sdf::Root::Load` and defaulted elements can be checked?